### ISSUE TITLE
11月11日发布的中间测试集在哪啊

### ISSUE THREAD

**Issue by gujianbo** [ChuXiaokai/WSDMCUP_BaiduPLM_Paddle#8]
Pre-training for Web Search比赛首页说的11月11日发布中间测试数据集，请问在哪可以找到啊
Intermediate test set release and intermediate submission starts.

**Comment by zoulixin93** [comment]
The intermediate test set is available at  https://aistudio.baidu.com/aistudio/datasetdetail/168759

**Comment by zoulixin93** [comment]
There is a demo code (https://aistudio.baidu.com/aistudio/projectdetail/4821122?contributionType=1 ) for submission.
Please refer to step 6 for submitting to the platform.
